.github/workflows/run_tests.yaml

```diff
@@ -5,20 +5,41 @@ run-name: Player And Plugin Tests
 on:
   pull_request:
     branches:
-      - "*"
+      - '*'
 
 jobs:
-  running-tests:
+  build:
```

Comment:
is it possible to call all of the jobs in one line?
something similar to what we do in run_canary.yaml:
```
running-tests:
    uses: kaltura/ovp-pipelines-pub/.github/workflows/player_tests.yaml@master
    with:
      node-version: '20.x'
      yarn-run-to-execute: 'build test type-check lint'
```